src/main/features/dashboard/views/macro/claimStatus/claimPartAdmitted.njk

```diff
@@ -0,0 +1,90 @@
+  {{ t('The defendant believes that they’ve paid the claim in full.') }}
+{% endmacro %}
+
+{% macro claimPartAdmittedAlreadyPaidForDefendantClaimDetails(claim) %}
+  <h2 class="heading-small">{{ t('Your response to the claim') }}</h2>
+  <p>
+    {{ t('We’ve emailed {{ claimantName }} telling them when and how you said you paid the claim.',
+      { claimantName: claim.claimData.claimant.name }) }}
+  </p>
+  <p>
+    {{ t('We’ll contact you to let you know how they respond. They can confirm you’ve paid and the claim is ' +
+      'settled, or they can proceed with it.') }}
+  </p>
+  <p>
+    {{ _downloadResponseLink(claim) }}
+  </p>
+{% endmacro %}
+
+{% macro claimPartAdmittedAlreadyPaidForClaimantClaimDetails(claim) %}
+  {% set paymentDate = claim.response.paymentDeclaration.paidDate | date %}
+
+  <h2 class="heading-small">{{ t('The defendant’s response') }}</h2>
+  <p>
+    {{ t('{{ defendantName }} says they paid you {{ amount }} on {{ paymentDate }}.',
+      { defendantName: claim.claimData.defendant.name, amount: claim.response.amount | numeral, paymentDate: paymentDate }) }}
+  </p>
+  <p> {{ t('You can accept or reject this response.') }}</p>
+  <p>
+    {{ t('You need to respond before 4pm on {{ respondDate}} or your claim won’t proceed.', {respondDate: claim.respondToResponseDeadline | date}) }}
+  </p>
+{% endmacro %}
+
+{% macro claimPartAdmittedToOweForDefendantDashboard(claim) %}
+  {{ t('You’ve admitted part of the claim.') }}
+{% endmacro %}
+
+{% macro claimPartAdmittedToOweForClaimantDashboard(claim) %}
+  {{ t('The defendant believes they owe you {{ amount }}. You can accept or reject that this is the amount owed.',
+    { amount: claim.response.amount | numeral }) }}
+{% endmacro %}
+
+{% macro claimPartAdmittedToOweForDefendantClaimDetails(claim) %}
+
+  {% if claim.response.paymentIntention.paymentOption === domain.PaymentOption.IMMEDIATELY %}
+    {% set paymentMethod = 'immediately' %}
+  {% elseif claim.response.paymentIntention.paymentOption === domain.PaymentOption.BY_SPECIFIED_DATE %}
+    {% set paymentMethod = 'on ' + claim.response.paymentIntention.paymentDate | date %}
+  {% elseif  claim.response.paymentIntention.paymentOption === domain.PaymentOption.INSTALMENTS %}
+    {% set repaymentPlan = claim.response.paymentIntention.repaymentPlan %}
+    {% set installmentAmount = repaymentPlan.instalmentAmount | numeral %}
+    {% set paymentSchedule = repaymentPlan.paymentSchedule | renderPaymentSchedule %}
+    {% set paymentMethod =  installmentAmount + ' ' + paymentSchedule + ' starting '+ repaymentPlan.firstPaymentDate |date %}
+  {% endif %}
+
+  <h2 class="heading-small">{{ t('Your response to the claim') }}</h2>
+  <p>
+    {{ t('You’ve said you owe {{ amount }} and offered to pay {{ claimantName }} {{ paymentMethod }}.',
+      { amount: claim.response.amount | numeral, claimantName: claim.claimData.claimant.name, paymentMethod: paymentMethod }) }}
```

Comment:
My understanding is that when we pass values to i18n placeholders they are not being translated and value is interpolated as is. That means that `'immediately'`, `'on ' + claim.response.paymentIntention.paymentDate | date` and  `installmentAmount + ' ' + paymentSchedule + ' starting '+ repaymentPlan.firstPaymentDate |date` won't get translated.